dev-docs/source/GettingStarted.rst

```diff
@@ -56,3 +54,104 @@ package and install it:
 
 where ``X.Y.Z`` should be replaced with the version that was downloaded.
 
+OSX
+---
+The build environment on OS X is described here :ref:`BuildingOnOSX`.
+
+Checking out the Mantid code
+############################
+We use `Git`_ as our version control system (VCS). The master copies of our repositories are located at `GitHub <http://github.com/mantidproject>`_. We have a number of repositories, of which the main one (the one containing all the source code for Mantid itself) is called simply `mantid <http://github.com/mantidproject/mantid>`_.
+
+Naturally, you will require git installed on your machine. Windows and Mac OSX downloads are available at https://www.git-scm.com/; Linux users should install via their package manager. This will give you access to git via the command line, with the Windows and Mac installers also including a simple graphical tool (Git Gui). Other graphical tools include `TortoiseGit <http://code.google.com/p/tortoisegit/>`_ (for windows) and a Visual Studio extension. There are others (do add your favourite!). Git is also available within Eclipse and QtCreator.
+
+Set up Git
+----------
```

Comment:
This should just point to external documentation, e.g. https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup

Reply:
Are you talking about the whole "Set up Git" section?

Reply:
Yep.